src/main/java/com/example/service/BookServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.example.service;
+
+import com.example.model.Book;
+import com.example.repository.BookRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class BookServiceImpl implements BookService {
+
```

Comment:
either remove empty line after class definition everywhere or keep it everywhere, consistency is key :)